package.json

```diff
@@ -291,10 +292,9 @@
     "jest-circus": "27.3.1",
     "jest-dev-server": "6.0.0",
     "msw": "0.31.0",
-    "playwright": "1.25.0",
-    "playwright-chromium": "1.25.0",
-    "playwright-core": "1.25.0",
-    "prettier": "2.7.1",
```

Comment:
what happened to prettier?

Reply:
I actually added in the other PR fixing imports, but it's not needed to be added explicitly, as it's a dependency of `@stacks/prettier-config`